### PR TITLE
adding cmake support and tests for openjpeg

### LINKED ISSUE #218: add cmake support and testing with openjpeg
add cmake support and testing with openjpeg

### PR DESCRIPTION
**Pull request by AlysonStahl-NOAA**
Fixes #218 
Part of #162

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "grb2.h": "c"
```
Comment (edwardhartnett) [inline .vscode/settings.json]: Is this file intentional?

Comment (AlysonStahl-NOAA) [reply .vscode/settings.json]: Nope, I'll remove it. Thanks for pointing it out.

**Review by edwardhartnett — APPROVED** [review]
